flocker/node/agents/blockdevice.py

```diff
@@ -371,14 +371,84 @@ def run(self, deployer):
         ).write(_logger)
         return succeed(None)
 
+# FLOC-1549
+
+# class IBlockDeviceAsyncAPI(Interface):
+#     """
+#     Common operations provided by all block devices backends, exposed via
+#     asynchronous methods.
+#     """
+#     def create_volume(dataset_id, size):
```

Comment:
I think this is sufficiently internal that we don't need to worry about confusing naive folks by using "volume" here.  Also, in context, "volume" makes a lot more sense because that's what they're mostly called on the backends this code is for interfacing with.  So I'll stick with volume unless you feel strongly.